src/main/java/mate/academy/dao/MovieDao.java

```diff
@@ -10,4 +10,8 @@ public interface MovieDao {
     Optional<Movie> get(Long id);
 
     List<Movie> getAll();
+
+    boolean update(Movie movie);
```

Comment:
Consider changing the return type of the `update` method to `Movie` or `void` to better reflect its operation. Returning a boolean might not provide enough information about the update process.